packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -504,13 +505,47 @@ describe("Garbage Collection Tests", () => {
             garbageCollector.nodeUpdated(nodes[3], "Loaded", Date.now(), testPkgPath);
             assert(
                 mockLogger.matchEvents([
-                    { eventName: changedEvent, timeout: deleteTimeoutMs, id: nodes[1], pkg: eventPkg },
-                    { eventName: changedEvent, timeout: deleteTimeoutMs, id: nodes[2], pkg: eventPkg },
-                    { eventName: loadedEvent, timeout: deleteTimeoutMs, id: nodes[3], pkg: eventPkg },
+                    { eventName: changedEvent, timeout: inactiveTimeoutMs, id: nodes[1], pkg: eventPkg },
+                    { eventName: changedEvent, timeout: inactiveTimeoutMs, id: nodes[2], pkg: eventPkg },
+                    { eventName: loadedEvent, timeout: inactiveTimeoutMs, id: nodes[3], pkg: eventPkg },
                 ]),
                 "inactiveObjectChanged event not generated as expected",
             );
         });
+
+        it("can override inactive timeout via feature flags", async () => {
+            const inactiveTimeoutOverrideMs = 100;
```

Comment:
Is there a way to assert that this override is less than the value passed in via options?  To protect against this test being invalidated if the value passed in changes.

Reply:
This value will override what ever is passed via options (less or more). Are you saying that we should validate the value is indeed overridden by the garbage collector?